src/routes/api/submit/photo/+server.ts

```diff
@@ -92,4 +92,4 @@ function validatePhoto(photo: File, fileType: string): boolean {
     if (fileType.match(/^(png|jpg|jpeg|gif|bmp|webp)$/) === null) return false
 
     return true
-}
+}
```

Comment:
?

Reply:
Not sure how this got here. I don't see the problem here though

Reply:
It could acutally just be a removed (or added) space